stanford_events_importer.module

```diff
@@ -511,3 +511,14 @@ function stanford_events_importer_preprocess_field(&$vars) {
     }
   }
 }
+
+/**
+* Implements hook_feeds_presave().
+* Set the body field input format to content_editor_text_format if the format is available.
+*/
+function stanford_events_importer_feeds_presave(FeedsSource $source, $entity, $item) {
+  $formats = filter_formats();
+  if(!empty($formats['content_editor_text_format'])) {
+    $entity->body['und'][0]['format'] = 'content_editor_text_format';
```

Comment:
This looks to be a good solution. A couple of small changes.

Check to see if the entity has a body field and that the entity type is of stanford_event.
When assigning the format to the body loop through all of the languages available as well as each field entry. We don't know for sure the body field only has one entry and that it is under no language specific format.